langserver/internal/refs/testdata/http-request-headers.go

```diff
@@ -0,0 +1,12 @@
+package main
+
```

Comment:
usually Go filenames have underscores, not dashes. it's a testdata file, so not a big deal.